### PR TITLE
Removing databinding

### PR DESCRIPTION
**Pull request by michaem**
Removed databinding by followin PRs from https://github.com/tinkoff-mobile-tech/forma

### CODE REVIEW

**Review by Jacks0N23 — DISMISSED** [review]

**Comment by stepango** [comment]
Could we clean up Readme.md too?

**Review by stepango — DISMISSED** [review]

File: plugins/android/src/main/java/dataBinding.kt [hunk 1]
```diff
@@ -1,123 +0,0 @@
...
-    owner: Owner = NoOwner,
-    visibility: Visibility = Public,
-    dependencies: FormaDependency = emptyDependency(),
-    consumerMinificationFiles: Set<String> = emptySet()
-) {
-    checkDataBindingFlag()
-
-    disallowResources()
-
-    target.validate(DataBindingAdapterTargetTemplate)
```
Comment (grine4ka) [inline plugins/android/src/main/java/dataBinding.kt:96]: I think that `DataBindingAdapterTargetTemplate` and `DataBindingTargetTemplate` should also be deleted

File: build-dependencies/dependencies/src/main/kotlin/ViewBinding.kt
Comment (grine4ka) [inline build-dependencies/dependencies/src/main/kotlin/ViewBinding.kt:1]: There is also file `Tools.kt` where old databinding versions are declared.
Should it be removed?

File: plugins/android/src/main/java/dataBinding.kt (hunk 1 above)
Comment (michaem) [reply plugins/android/src/main/java/dataBinding.kt:96]: Done

**Review by stepango — DISMISSED** [review]

**Review by stepango — APPROVED** [review]

**Pull request merged**